avaframe/com1DFAPy/com1DFA.py

```diff
@@ -569,6 +565,10 @@ def initializeParticles(cfg, relRaster, dem, logName=''):
         # plt.plot(particles['y'], particles['z'], 'k+')
         # plt.show()
     else:
+        # initiate random generator
+        seed = 12345
```

Comment:
This is a setting, please move to a dev flag at top of file or to the ini file...

Reply:
done